packages/anoncreds-rs/package.json

```diff
@@ -32,15 +32,15 @@
     "tsyringe": "^4.8.0"
   },
   "devDependencies": {
-    "@hyperledger/anoncreds-nodejs": "^0.2.0-dev.4",
-    "@hyperledger/anoncreds-shared": "^0.2.0-dev.4",
+    "@hyperledger/anoncreds-nodejs": "^0.2.0-dev.5",
+    "@hyperledger/anoncreds-shared": "^0.2.0-dev.5",
     "@types/ref-array-di": "^1.2.6",
     "@types/ref-struct-di": "^1.1.10",
     "reflect-metadata": "^0.1.13",
     "rimraf": "^4.4.0",
     "typescript": "~4.9.5"
   },
   "peerDependencies": {
-    "@hyperledger/anoncreds-shared": "^0.2.0-dev.4"
+    "@hyperledger/anoncreds-shared": "^0.2.0-dev.5"
```

Comment:
Yes we make shared a peer dep, as we don't want to force which version you install, as then it can lead to inconsistencies if you install react-native/node yourself. By making it a peer dep, once you install react-native/node you automatiaclly have shared with the correct version